spec/controllers/concerns/shared_examples_for_session_person_creator.rb

```diff
@@ -57,11 +57,13 @@
 
       it_behaves_like "existing person returned"
 
+      # rubocop:disable RSpec/NoExpectationExample
       it "extracts email from auth hash" do
```

Comment:
There's no expectation, so nothing is being tested here. It should either be changed to you an expectation, or removed.